css/main.css

```diff
@@ -27,30 +27,102 @@ header {
   color: white;
 }
 
-#desktop-nav, #desktop-nav ul {
+li {
+  list-style: none;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  color: #fff;
+  padding: 5px;
+}
+
+.full-screen {
+  height: 100vh;
+  display: block;
+}
+
+#nav {
+  width: 100%;
+  height: 100%;
+}
+
+.logo-menu-container {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
   display: none;
 }
 
-#logo {
+.logo.active {
+  display: block;
   font-family: "Roboto", sans-serif;
   font-style: normal;
   font-weight: bold;
   font-size: 18px;
   line-height: 24px;
 }
 
-.menu-bars {
+.nav-menu {
+  display: none;
+}
+
+.nav-menu.active {
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.nav-menu li {
+  display: flex;
+  border-bottom: rgba(245, 245, 245, 0.511) solid 1px;
+  width: 100%;
+}
+
+.nav-menu li a {
+  text-decoration: none;
+  color: whitesmoke;
+  margin-right: auto;
+}
+
+.menu-button {
   background: transparent;
   border: none;
   outline: none;
 }
 
-.menu-bars i {
+.menu-button.active {
+  margin-left: auto;
+}
+
+.menu-button i {
+  display: none;
+  width: 18px;
+  height: 12px;
+  color: white;
+}
+
+.menu-button i.active {
+  display: block;
   width: 18px;
   height: 12px;
   color: white;
 }
 
+.x-icon {
+  font-size: 4.5rem;
+}
```

Comment:
Kindly adjust your X button to fit with the Figma design.

Figma design:
![design_menu](https://user-images.githubusercontent.com/42799579/190256015-96ecaa7d-2401-4864-ad87-8dadd8f805b3.png)

Your design:
![user_menu](https://user-images.githubusercontent.com/42799579/190255990-9aefb672-1189-44cf-a5b3-95cdca484631.png)